yb-voyager/cmd/export.go

```diff
@@ -53,14 +53,14 @@ func registerCommonExportFlags(cmd *cobra.Command) {
 		"source database server host")
 
 	cmd.Flags().IntVar(&source.Port, "source-db-port", -1,
-		"source database server port number")
+		"source database server port number.\nDefault: Oracle(1521), MySQL(3306), PostgreSQL(5432)")
 
 	cmd.Flags().StringVar(&source.User, "source-db-user", "",
 		"connect to source database as specified user")
 
 	// TODO: All sensitive parameters can be taken from the environment variable
 	cmd.Flags().StringVar(&source.Password, "source-db-password", "",
-		"connect to source as specified user")
+		"source password to connect to the specified user")
```

Comment:
source password to connect [using/as] the specified user

Reply:
done